vcsh

```diff
@@ -277,6 +277,13 @@ list() {
 	done
 }
 
+list_where_remote() {
+	for VCSH_REPO_NAME in $(list); do
+		GIT_DIR=$VCSH_REPO_D/$VCSH_REPO_NAME.git; export GIT_DIR
+		git config remote.origin.url >/dev/null && echo "$VCSH_REPO_NAME"
```

Comment:
What if `origin` isn't the remote? I realize `vcsh clone` sets that up by default but I have some systems where it's been changed, this would cause basic functions to fail. Right now the only place where that is hard coded is `clone()` and I think it's probably useful to keep it that way.

Reply:
good point, I changed it to simply check for any existing remote. whatever the repo has set up as default remote should be used then, and if there's no remote it'll be ignored by vcsh's push operation.